### ISSUE TITLE
ArrayIndexOutOfBoundsException while interacting with Exif information

### ISSUE THREAD

**Issue by xrigau** [onfido/onfido-android-sdk#215]
#### What was the expected behaviour?

The app shouldn't crash, the Onfido document capture should take the photo and proceed to the next step.

#### What happened instead?

The app crashes in the Document Capture step Confirmation of the Onfido flow.

#### Version info:

- `onfido-capture-sdk` version: `com.onfido.sdk.capture:onfido-capture-sdk-core:12.3.0` - Notice we're using `onfido-capture-sdk-core`
- Android API version(s): According to Firebase, this crash has only appeared in Android 9.

- `compileSdkVersion`: 33
- `targetSdkVersion`: 31
- `minSdkVersion`: 27
- Support library version (if applicable): N/A

#### Steps to reproduce:

We don't know, we haven't been able to reproduce this issue.

#### Onfido Flow initialization code:

```kotlin
val tokenExpirationHandler = // ...
    val config = OnfidoConfig.builder(context)
        .withSDKToken("token", tokenExpirationHandler)
        .build()
    val intent = onfido.createIntent(config)
    startActivity(intent)
```

#### Issue severity information (number of users/crashes if applicable):

This issue has 19 crash events affecting 5 users for the past 2 weeks.

#### Additional LogCat output:

Stacktrace from Firebase:

```
Fatal Exception: io.reactivex.rxjava3.exceptions.UndeliverableException: The exception could not be delivered to the consumer because it has already canceled/disposed the flow or the exception has nowhere to go to begin with. Further reading: https://github.com/ReactiveX/RxJava/wiki/What's-different-in-2.0#error-handling | java.lang.ArrayIndexOutOfBoundsException: length=0; index=0
       at io.reactivex.rxjava3.plugins.RxJavaPlugins.onError(RxJavaPlugins.java:372)
       at io.reactivex.rxjava3.internal.observers.ConsumerSingleObserver.onSuccess(ConsumerSingleObserver.java:65)
       at io.reactivex.rxjava3.internal.operators.single.SingleObserveOn$ObserveOnSingleObserver.run(SingleObserveOn.java:81)
       at io.reactivex.rxjava3.android.schedulers.HandlerScheduler$ScheduledRunnable.run(HandlerScheduler.java:123)
       at android.os.Handler.handleCallback(Handler.java:873)
       at android.os.Handler.dispatchMessage(Handler.java:99)
       at android.os.Looper.loop(Looper.java:193)
       at android.app.ActivityThread.main(ActivityThread.java:6852)
       at java.lang.reflect.Method.invoke(Method.java)
       at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:504)
       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:858)

aused by java.lang.ArrayIndexOutOfBoundsException: length=0; index=0
       at android.media.ExifInterface.guessDataFormat(ExifInterface.java:3670)
       at android.media.ExifInterface.setAttribute(ExifInterface.java:1547)
       at com.onfido.android.sdk.capture.ui.camera.Exif.copy(Exif.java:92)
       at com.onfido.android.sdk.capture.utils.ImageUtils.cropImage$onfido_capture_sdk_core_release(ImageUtils.java:142)
       at com.onfido.android.sdk.capture.ui.camera.CaptureActivity.uploadDocument(CaptureActivity.java:14)
       at com.onfido.android.sdk.capture.ui.camera.CaptureActivity.uploadImage(CaptureActivity.java:38)
       at com.onfido.android.sdk.capture.ui.camera.CapturePresenter.onPostCaptureValidationsFinished(CapturePresenter.java:34)
       at com.onfido.android.sdk.capture.ui.camera.CapturePresenter.applyPostCaptureValidations$lambda-36(CapturePresenter.java:15)
       at com.onfido.android.sdk.capture.ui.camera.CapturePresenter.$r8$lambda$Upakdy4SOE8NjYVZvqpM-WTN6Zo(CapturePresenter.java)
       at com.onfido.android.sdk.capture.ui.OnfidoPresenter$$InternalSyntheticLambda$0$ef40f30172e71dc6324c4c15089127c09f65f9a623111a3df20ac0736e53c110$0.accept$bridge(OnfidoPresenter.java:86)
       at io.reactivex.rxjava3.internal.observers.ConsumerSingleObserver.onSuccess(ConsumerSingleObserver.java:62)
       at io.reactivex.rxjava3.internal.operators.single.SingleObserveOn$ObserveOnSingleObserver.run(SingleObserveOn.java:81)
       at io.reactivex.rxjava3.android.schedulers.HandlerScheduler$ScheduledRunnable.run(HandlerScheduler.java:123)
       at android.os.Handler.handleCallback(Handler.java:873)
       at android.os.Handler.dispatchMessage(Handler.java:99)
       at android.os.Looper.loop(Looper.java:193)
       at android.app.ActivityThread.main(ActivityThread.java:6852)
       at java.lang.reflect.Method.invoke(Method.java)
       at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:504)
       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:858)
```

#### Other useful info:

Thanks to the analytics hook, we have the following details with this crash report:

```
mode: {"orientation":"PORTRAIT"}
country: [Many - we operate in KE, UG, GH and NG]
name: DOCUMENT_CAPTURE_CONFIRMATION_FRONT
doc_type: [national_id, passport, and probably others]
firebase_screen_class: CaptureActivity
is_autocapture: false
```

**Comment by xrigau** [comment]
Hey @Tobi0077 @gurpreet-onfido from the release notes of the latest release ([13.0.1](https://github.com/onfido/onfido-android-sdk/releases/tag/13.0.1)), it seems this should've been fixed, correct?

**Comment by gurpreet-onfido** [comment]
Hello @xrigau Yes, my team has confirmed that it is fixed in the release 13.0.1. Please let me know if there is anything else I can help you with.

Kind regards
Gurpreet Singh

**Comment by xrigau** [comment]
Great, thank you!
